assets/terraform/libvirt/cloudinit/ubuntu.cfg

```diff
@@ -0,0 +1,68 @@
+#cloud-config
```

Comment:
For Ubuntu 16.04 on libvirt I ran into problem with the same `ubuntu` hostname for every node and gravity fails to install because of that. Can you add check that and add `hostname` module in cloud-init if needed?